src/bpmn-properties-panel/icons/index.js

```diff
@@ -1,22 +1,175 @@
-import CallActivityIcon from './CallActivity.svg';
-import EndEventIcon from './EndEvent.svg';
-import ExclusiveGatewayIcon from './ExclusiveGateway.svg';
-import ReceiveTaskParallelIcon from './ReceiveTaskParallel.svg';
-import ServiceTaskIcon from './ServiceTask.svg';
-import StartEventIcon from './StartEvent.svg';
-import SubProcessCollapsedIcon from './SubProcessCollapsed.svg';
-import UserTaskIcon from './UserTask.svg';
+import BusinessRuleTaskIcon from './bpmn-icon-business-rule-task.svg';
```

Comment:
This file is really blown up by the number of icons. For now, I can't see real alternatives, because we have to very explicit.